Assets/Scripts/Managers/SFXManager.cs

```diff
@@ -0,0 +1,60 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.SceneManagement;
+
+public class SFXManager : MonoBehaviour
+{
+    private AudioSource audioSource;
+    public static SFXManager instance;
+    [HideInInspector]
+    public bool isSettings = false;
+    [HideInInspector]
+    public bool isPlanet = false;
+
+    private void Start()
+    {
+        DontDestroyOnLoad(this.gameObject);
+        audioSource = GetComponent<AudioSource>();
+        SetVolume();
+        if(SceneManager.GetSceneByName("Settings").isLoaded)
+        { //gets the settings scene and checks if its loaded
+            isSettings = true;
+        }
+        else if(SceneManager.GetSceneByName("Planet").isLoaded)
+        { //gets the settings scene and checks if its loaded
```

Comment:
I assume this comment should say planet instead of Settings

Reply:
Yup. Fixed.